products/sle12/profiles/stig.profile

```diff
@@ -246,7 +246,11 @@ selections:
     - sssd_offline_cred_expiration
     - sudo_remove_no_authenticate
     - sudo_remove_nopasswd
+<<<<<<< HEAD
```

Comment:
Probably leftover from rebase, please fix it.

Reply:
Thanks :bow: @vojtapolasek fixed that and rebase once again